src/material-experimental/mdc-slider/slider-thumb-decorator.html

```diff
@@ -0,0 +1,10 @@
+<ng-container *ngIf="isDiscrete">
```

Comment:
I think you can avoid an extra level of nesting by setting the `*ngIf` on the `mdc-slider__value-indicator-container`.